client/state/terms/reducer.js

```diff
@@ -103,17 +103,14 @@ export const queries = createReducer(
 			} );
 		},
 		[ DESERIALIZE ]: state => {
-			if ( ! isValidStateWithSchema( state, queriesSchema ) ) {
-				return {};
-			}
-
```

Comment:
For this one, we'd want to keep this and mark with `queries.hasCustomPersistence = true;`, right?

Reply:
yep, will do.